Xcode Templates/CyanicComponent.xctemplate/___FILEBASENAME___Component.swift

```diff
@@ -0,0 +1,39 @@
+import Cyanic
+import LayoutKit
+import UIKit
+
+// sourcery: AutoEquatableComponent,AutoHashableComponent
+// sourcery: Component = ___VARIABLE_productName:identifier___Component
+public protocol ___VARIABLE_productName:identifier___ComponentType: StaticHeightComponent {
+
+    // define here your component properties and annotations and run sourcery
+    // sourcery: defaultValue = "UIColor.clear"
+    var backgroundColor: UIColor { get set }
+
+}
+
+// sourcery: AutoGenerateComponent,AutoGenerateComponentExtension
+// sourcery: ComponentLayout = ___VARIABLE_productName:identifier___ComponentLayout
+public struct ___VARIABLE_productName:identifier___Component: ___VARIABLE_productName:identifier___ComponentType {
+    // left initially empty
+}
+
+public final class ___VARIABLE_productName:identifier___ComponentLayout: SizeLayout<UIView>, ComponentLayout {
+
+    public init(component: ___VARIABLE_productName:identifier___Component) {
+        let size: CGSize = CGSize(width: component.width, height: component.height)
+        super.init(
+            minWidth: size.width,
+            maxWidth: size.width,
+            minHeight: size.height,
+            maxHeight: size.height,
+            alignment: Alignment.center,
+            flexibility: Flexibility.inflexible,
```

Comment:
Flexibility should be flexible by default. In general, it will arrange the views normally, in cases where nested components are used, you want it to be flexible.